lambda/writePass/index.js

```diff
@@ -520,6 +521,8 @@ exports.handler = async (event, context) => {
             Authorization: process.env.GC_NOTIFY_API_KEY,
             'Content-Type': 'application/json'
           },
+          maxContentLength: Infinity,
```

Comment:
Reason for this too? I think Default for both of these is `-1`. Works for me without this change

Reply:
I'll remove this. Axios gave me some errors before but now that I think about it might have just been a local problem.